mount/mount_linux.go

```diff
@@ -303,7 +303,7 @@ func longestCommonPrefix(strs []string) string {
 	}
 
 	// find out the common part between min and max
-	for i := 0; i < len(min) && i < len(max); i++ {
```

Comment:
len(min) can be larger than len(max), like min=abc, max=b. but if len(min) != len(max), min[i] != max[i] and i < len(min) and i < len(max). Just for readable, i should be `< len(min) && < len(max)`

Reply:
yeah you're right !   I have some other considerations, just for a talk:

If x > y (alphabetic order), we have two possibility:
  1. x and y have common prefix,  
      1.1  if len(x) >= len(y), like x = ab**  y =ab*, so just I < len(y) is ok
      1.2  If len(x) < len(y), like x = ab*  y =ab**,   there must be a character after  x’s   and  y’s prefix,
           x[2] > y[2], so the loop will end

  2. If x and y don’t have common prefix, is just same as 1.2

Reply:
Sorry for late reply. @ide-rea your explanation is correct. Could you mind to update the comment? thanks

Reply:
OK, I'll do this